policies.tf

```
@@ -0,0 +1,70 @@
+# The policy document that allows assumption of all roles needed in order to
+# manage IAM users and groups.
+data "aws_iam_policy_document" "iam_manager_roles" {
+  statement {
+    actions = [
+      "sts:AssumeRole",
+      "sts:TagSession",
+    ]
+
+    effect = "Allow"
+
+    resources = local.required_roles
+
+    sid = "AllowRequiredRoles"
```

Something like this seems more in-line with this document.

```suggestion
    sid = "AssumeRequiredRoles"
```